blivet/devices/lvm.py

```diff
@@ -2167,6 +2302,19 @@ def resize(self):
         udev.settle()
         blockdev.lvm.lvresize(self.vg.name, self._name, self.size)
 
+    @type_specific
+    def _add_log_vol(self, lv):
+        pass
+
+    @type_specific
+    def _remove_log_vol(self, lv):
+        pass
+
+    @property
+    @type_specific
+    def lvs(self):
+        return []
+
```

Comment:
Same here, more or less... why is this needed if it wasn't needed for thinp?

Reply:
Because before this thinpool was the only "type" that needed these properties so these could be part of the thinpool "interface". Now we have two types that need it, so these need to be part of the mixin interface.